Formula/prs.rb

```diff
@@ -0,0 +1,33 @@
+class Prs < Formula
+  desc "Secure, fast & convenient password manager CLI with GPG & git sync"
+  homepage "https://timvisee.com/projects/prs"
+  url "https://github.com/timvisee/prs/archive/v0.4.1.tar.gz"
+  sha256 "f7f8f5d815cf1c4034f1c2aa36ed29b7e3ab2a884791b4b69ffc845d5d111524"
+  license "GPL-3.0-only"
+
+  depends_on "rust" => :build
+  depends_on "gpgme"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "libxcb"
+    depends_on "openssl@3"
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "./cli")
+
+    system bin/"prs", "internal", "completions", "--output=completions/", "bash", "fish", "zsh"
+    bash_completion.install "completions/prs.bash"
+    fish_completion.install "completions/prs.fish"
+    zsh_completion.install "completions/_prs"
+  end
+
+  test do
+    system bin/"prs", "help"
+    system bin/"prs"
+    system bin/"prs", "--store=.pass", "init"
+    system bin/"prs", "--store=.pass", "list"
+    assert_equal "", shell_output("#{bin}/prs --store=.pass --quiet list")
```

Comment:
```suggestion
    ENV["PASSWORD_STORE_DIR"] = testpath/".store"
    expected = <<~EOS
      Now generate and add a new recipient key for yourself:
          prs recipients generate
    EOS
    assert_equal expected, shell_output("#{bin}/prs init --no-interactive")

    assert_match "prs-cli #{version}", shell_output("#{bin}/prs --version")
```

Reply:
The output of `prs init` will change in new versions. Is it still a good idea to assert this then?

Asserting `--version` output seems like a good idea.

Reply:
> The output of `prs init` will change in new versions. Is it still a good idea to assert this then?

this is only for the CI run? Technically, we should always assert the output.

Reply:
> assert_equal "", shell_output("#{bin}/prs --store=.pass --quiet list")

This actually does not quite make sense, as it would always pass when checking against `""`

Reply:
> This actually does not quite make sense, as it would always pass when checking against ""

Why does that not make sense?

`--quiet` does not mean it never outputs. It means extra information such as a header or store root is omitted from the output.

Reply:
You're confusing `assert_match` @chenrui333 that would always pass since it would mean an empty regex